breakout_1_Introduction-to-Grafana-Observability/breakout_1_answers/3.8-k8s-o11y.md

```diff
@@ -8,10 +8,12 @@ To find this:
 1. Expand the Infrastructure and then Kubernetes menu
 1. Click on Cost
 
-    ![Clusters](/images/breakout_1/3.8-k8s-olly-1.png)
+    ![Clusters](/images/breakout_1/3.8-k8s-o11y-1.png)
 
 1. Even though there is only one k8s cluster, let's make sure we are focused on that. In the cluster variable input, change it from `All` to the k8s cluster
   
-    ![Clusters](/images/breakout_1/3.8-k8s-olly-2.png)
+    ![Clusters](/images/breakout_1/3.8-k8s-o11y-2.png)
 
 1. In this example we can see last month cost us around $72 and it's expected to be the same next month
```

Comment:
@Alex3k Seems like new appenvs now struggle with this one? https://badf52.grafana.net/a/grafana-k8s-app/cost?from=now-24h&to=now&timezone=browser&var-cluster=badf52-cluster&refresh=1m